frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -70,9 +120,43 @@ const TableList: FC<{}> = () => {
       dataIndex: 'attachments',
       render: (attachments: []) => attachments.length,
     },
+    {
+      title: formatMessage({ id: localeKeys.lists.actions }),
+      dataIndex: 'id',
+      render: (_, { id, referenceNumber }: Letter) => (
```

Comment:
W projekcie jest gdzieś lodash, proponuję jakąś inną nazwę zmiennej zamiast `_`. Jakąkolwiek. Np. `x`.

Reply:
Underscore jest ignorowany przez prettiera, przez co nie rzuca błędów unused variable.

Reply:
Dokładnie tak. To dość powszechna konwencja  dla nazwania nie używanych parametrów funkcji, nawet była gdzieś użyta w projekcie ;)